src/registry/checkbox/edit.tsx

```diff
@@ -155,4 +155,33 @@ const DefaultValue: React.FC<DefaultValueProps> = ({description}) => {
   );
 };
 
+const renderComparisonValueInput: React.FC<ComparisonValueProps> = props => {
+  return (
+    <Select
+      {...props}
+      options={[
+        {
+          value: true,
+          label: (
+            <FormattedMessage
+              description="Label of the 'true' checkbox option"
+              defaultMessage="Checked"
+            />
+          ),
+        },
+        {
+          value: false,
+          label: (
+            <FormattedMessage
+              description="Label of the 'false' checkbox option"
+              defaultMessage="Not checked"
+            />
+          ),
+        },
+      ]}
+    />
+  );
+};
+
```

Comment:
Would a checkbox component be better suited? Or did you have to use a select as the `Has the value` label is not customizable (well not anymore with your other changes), so it would look weird with a checkbox

Reply:
I thought that it looked a bit weird the 'has the value' with the checkbox, that's why I went for the dropdown :thinking: